src/main/java/de/chojo/repbot/commands/Scan.java

```diff
@@ -319,7 +319,7 @@ public boolean scan() {
 
                 if (message.getAuthor().isBot()) continue;
 
-                var result = messageAnalyzer.processMessage(pattern, message, 0, false, 0.85, 3);
+                var result = messageAnalyzer.processMessage(pattern, message,null, false, 0.85, 3);
```

Comment:
Missing space between the second and third argument

Reply:
Changed UwU